src/scss/hpinc/_hpinc.defaults.scss

```diff
@@ -23,7 +24,7 @@ $focus-border-color: #99d5ef;
 
 html,
 .brand-font {
```

Comment:
I think the .brand-font selector should be added to _base.font.scss and we can remove this set of selectors from this file.